spec/features/position_creation_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe 'creating new positions' do
+  let!(:department) { create :department }
+  before :each do
+    when_current_user_is :staff, integration: true
+    visit new_position_path
+  end
+
+  context 'required fields are filled in' do
+    before :each do
+      within 'form.new_position' do
+        fill_in_fields_for Position, attributes:
+        { name: 'Our new shiny position',
+          default_interview_location: 'UMTS', department: department }
```

Comment:
If we're going to line-split this, it's probably most readable to have one attribute-value pair per line.